src/models/helpers/escapeFields.js

```diff
@@ -0,0 +1,52 @@
+import { escape } from 'lodash';
+import { auditLogger } from '../../logger';
+import safeParse from './safeParse';
+
```

Comment:
The actual new code from this PR: a function to escape a given field, or a given field within a data JSONB object